neuro_flow/config_loader.py

```diff
@@ -255,6 +255,7 @@ async def _tarball_from_github(
             async with self._github_session.get(
                 url=f"https://api.github.com/repos/{repo}/tarball/{version}"
             ) as response:
+                response.raise_for_status()
```

Comment:
It's a good idea to pass any error in general, but for 404 I personally prefer a custom error message.
```suggestion
                if response.status == HTTPNotFound.status_code:
                    raise ValueError(f"Cannot fetch action: either repository {repo} or tag {version} does not exists")
                response.raise_for_status()
```

Reply:
By the way, using branch name instead of tags wasn't supposed initially, but I have nothing against it.

Reply:
> custom error message

That sounds reasonable, thanks!

> By the way, using branch name instead of tags wasn't supposed initially, but I have nothing against it.

Didn't know that, tbh 😄